transport/src/androidUnitTest/kotlin/transport/core/AttachmentHandlerTest.kt

```diff
@@ -65,10 +67,14 @@ internal class AttachmentHandlerTest {
 
     @ExperimentalCoroutinesApi
     private val dispatcher: CoroutineDispatcher = UnconfinedTestDispatcher()
+    private val mockTracingProvider: TracingIdProvider = mockk {
+        every { getTracingId() } returns "test-tracing-id"
```

Comment:
- Why AttachmentHandler and rest of the handlers/providers get the TracingIdProvider in their constructor? 
- Use TestValues to return tracingId for tests.